gecko-media v0.1.0 build failed in Ubuntu 16.04

#19252 did not fix it.
```
sudo apt install libpulse-dev
```

Log:
```
tigercosmos@cook-190:~/servo$ ./mach build -d
   Compiling gecko-media v0.1.0 (https://github.com/servo/gecko-media.git#4417372d)
   Compiling hyper-openssl v0.2.6
   Compiling influent v0.4.0
   Compiling webdriver v0.32.0
error: failed to run custom build command for `gecko-media v0.1.0 (https://github.com/servo/gecko-media.git#4417372d)`
process didn't exit successfully: `/home/tigercosmos/servo/target/debug/build/gecko-media-ced88898ad69a8fd/build-script-build` (exit code: 101)
--- stdout
running: "cmake" "/home/tigercosmos/servo/.cargo/git/checkouts/gecko-media-6bedb392a81eb7cf/4417372/." "-DCMAKE_INSTALL_PREFIX=/home/tigercosmos/servo/target/debug/build/gecko-media-f56d23f3ebe44183/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_BUILD_TYPE=Debug"
-- Configuring done
-- Generating done
-- Build files have been written to: /home/tigercosmos/servo/target/debug/build/gecko-media-f56d23f3ebe44183/out/build
running: "cmake" "--build" "." "--target" "install" "--config" "Debug" "--"
[  8%] Built target stagefright
[  9%] Built target ffvpx
[ 10%] Built target ffmpeg57
[ 11%] Built target speex_resampler
[ 12%] Built target tests
[ 13%] Built target cubeb
[ 13%] Built target ogg
[ 46%] Built target opus
[ 86%] Built target gecko_core
[ 91%] Built target vorbis
[ 96%] Built target theora
[ 99%] Built target soundtouch
[100%] Built target gecko_media_cmake
Install the project...
-- Install configuration: "Debug"
-- Up-to-date: /home/tigercosmos/servo/target/debug/build/gecko-media-f56d23f3ebe44183/out/libgecko_media_cmake.a
cargo:root=/home/tigercosmos/servo/target/debug/build/gecko-media-f56d23f3ebe44183/out
cargo:rustc-link-search=native=/home/tigercosmos/servo/target/debug/build/gecko-media-f56d23f3ebe44183/out
cargo:rustc-link-lib=static=gecko_media_cmake
cargo:rustc-link-lib=stdc++
cargo:rustc-link-lib=dl

--- stderr
thread 'main' panicked at 'Unable to find libclang: "couldn\'t find any of [\'libclang.so\', \'libclang.so.*\'], set the LIBCLANG_PATH environment variable to a path where one of these files can be found (skipped: [])"', /checkout/src/libcore/result.rs:906:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

@nox  @philn 
after installing all mentioned above still get failed..
```
--- stderr
gecko/glue/include/GeckoMedia.h:10:10: fatal error: 'stddef.h' file not found
gecko/glue/include/GeckoMedia.h:10:10: fatal error: 'stddef.h' file not found, err: true
thread 'main' panicked at 'Unable to generate bindings: ()', /checkout/src/libcore/result.rs:906:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

--- a/README.md
+++ b/README.md
@@ -56,8 +56,10 @@ sudo apt install git curl freeglut3-dev autoconf libx11-dev \
     libfreetype6-dev libgl1-mesa-dri libglib2.0-dev xorg-dev \
     gperf g++ build-essential cmake virtualenv python-pip \
     libssl1.0-dev libbz2-dev libosmesa6-dev libxmu6 libxmu-dev \
-    libglu1-mesa-dev libgles2-mesa-dev libegl1-mesa-dev libdbus-1-dev \
-    libpulse-dev
+    libglu1-mesa-dev libgles2-mesa-dev libegl1-mesa-dev \
+    pulseaudio dbus-x11 libavcodec-dev libavformat-dev \
+    libavutil-dev libswresample-dev  libswscale-dev libdbus-1-dev \
+    libpulse-dev clang
 ```
 
 If you using a version prior to **Ubuntu 17.04** or **Debian Sid**, replace `libssl1.0-dev` with `libssl-dev`.

